### ISSUE TITLE
SSL Certificate Expired

### ISSUE THREAD

**Issue by jwagnerLegacy** [citruspi/Spotify-Notifications#111]
Just wanted to let you know that on the [download](https://downloads.spotify-notifications.citruspi.io/0.6.0-release.zip) page, the SSL certificate is expired.

**Comment by suschizu** [comment]
that's right @jwagnerLegacy - @citruspi - could you please renew your SSL-certificate for 
https://downloads.spotify-notifications.citruspi.io/0.6.0-release.zip ? thanks :)

**Comment by core-code** [comment]
still expired

**Comment by suschizu** [comment]
@citruspi - could you please fix your SSL-certificate for the latest release?

**Comment by nessup** [comment]
@citruspi can you please fix? :)
